README.md

```diff
@@ -58,24 +101,55 @@ To run the default script ('LTheory'), or
 
 - `python configure.py run <script_name_without_extension>`
```

Comment:
Above, it says "If the compilation is successful" under "Running a Lua App".

I suggest changing this to "By the time you reach this step, you now have..."